### ISSUE TITLE
not able to scroll previews left/right in any picker

### ISSUE THREAD

**Issue by feakuru** [nvim-telescope/telescope.nvim#3110]
### Description

hi! i am unable to use the shortcuts described in the README, specifically:
```
<C-f> 	Scroll left in preview window
<C-k> 	Scroll right in preview window
```
in any picker. Pickers that I checked are these:
```
      local builtin = require 'telescope.builtin'
      vim.keymap.set('n', '<leader>sf', builtin.find_files, { desc = '[S]earch [F]iles' })
      vim.keymap.set('n', '<leader>sw', builtin.grep_string, { desc = '[S]earch current [W]ord' })
      vim.keymap.set('n', '<leader>sg', builtin.live_grep, { desc = '[S]earch by [G]rep' })
      vim.keymap.set('n', '<leader><leader>', builtin.buffers, { desc = '[ ] Find existing buffers' })
```
upon further investigation, the described shortcuts are not appearing when using the `?` key from normal mode or `<C-/>` from insert. and they are not present on the actions module when trying to interact with it from lua.

has there been a decision to remove these from the release or are they new or something? sorry if i missed anything

### Neovim version

```markdown
NVIM v0.9.5
Build type: Release
LuaJIT 2.1.1713773202

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

### Operating system and version

Arch Linux

### Telescope version / branch / rev

0.1.x (commit 6312868)

### checkhealth telescope

```markdown
==============================================================================
telescope: require("telescope.health").check()

Checking for required plugins ~
- OK plenary installed.
- OK nvim-treesitter installed.

Checking external dependencies ~
- OK rg: found ripgrep 14.1.0
- OK fd: found fd 10.1.0

===== Installed extensions ===== ~

Telescope Extension: `fzf` ~
- OK lib working as expected
- OK file_sorter correctly configured
- OK generic_sorter correctly configured

Telescope Extension: `ui-select` ~
- No healthcheck provided
```

### Steps to reproduce

my config is at https://github.com/feakuru/nvim-config, but it's basically Kickstart with some light modifications, telescope is installed by Lazy with `branch = '0.1.x'`

### Expected behavior

being able to do things that are specified in the readme

### Actual behavior

not able to scroll previews + the action seems like it's missing from the plugin despite the readme

### Minimal config

```Lua
my config is at https://github.com/feakuru/nvim-config, but it's basically Kickstart with some light modifications, telescope is installed by Lazy with `branch = '0.1.x'`
```

**Comment by jamestrew** [comment]
This feature is only available in nightly telescope at the moment.

**Comment by ildar-shaymukhametov** [comment]
@jamestrew Is this feature available? I have the same problem (Neovim 0.10.1, Telescope 0.1.8)

**Comment by jamestrew** [comment]
Still only on master branch. It won't become part of stable telescope until telescope 0.2.0.
